config.tmpl

```diff
@@ -2,6 +2,7 @@
     "tokens": {
         "AccessToken": "${lacework_access_token}"
     },
+    %{ if lacework_agent_autoupgrade != true }"autoupgrade": "disable",%{ endif }
```

Comment:
This logic can be simplified by [using the unary operator](https://www.terraform.io/docs/language/expressions/operators.html)
```suggestion
    %{ if ! lacework_agent_autoupgrade }"autoupgrade": "disable",%{ endif }
```